langlib/lang.error/src/main/ballerina/error.bal

```diff
@@ -91,6 +111,10 @@ public isolated function stackTrace(error e) returns StackFrame[] {
 # The details of the conversion are specified by the ToString abstract operation
 # defined in the Ballerina Language Specification, using the direct style.
 #
+# ```
+# error("invalid salary", value = 0d).toString() ⇒ error("invalid salary",value=0)
```

Comment:
```suggestion
# error("invalid salary", value = -1000).toString() ⇒ error("invalid salary",value=-1000)
```

`0` is not a invalid value for a salary?

Reply:
Why not? Depends on our definition (e.g., minimum salary is 100)?

Reply:
Ok, lets go with `0`.